docs/proposals/ssh_adapter.md

```diff
@@ -0,0 +1,194 @@
+# SSH protocol proposal
+
+We'd like to implement a protocol for Git LFS that uses SSH protocol
```

Comment:
I think that it was good that you mentioned in your commit message that this proposal is an aspiration, and not a promise that we will implement it, or a requirement of the same to anyone else. Do you think that it would be worthwhile to mention that here, too?

Reply:
Sure, I can do that.